port_forwarding.go

```diff
@@ -99,10 +99,25 @@ func exposeHostPorts(ctx context.Context, req *ContainerRequest, ports ...int) (
 		return sshdConnectHook, fmt.Errorf("new sshd container: %w", err)
 	}
 
-	// IP in the first network of the container
-	sshdIP, err := sshdContainer.ContainerIP(context.Background())
+	// IP in the first network of the container.
+	inspect, err := sshdContainer.Inspect(ctx)
 	if err != nil {
-		return sshdConnectHook, fmt.Errorf("get sshd container IP: %w", err)
+		return sshdConnectHook, fmt.Errorf("inspect sshd container: %w", err)
+	}
+
+	sshdIP := inspect.NetworkSettings.IPAddress
```

Comment:
follow-up: this will be simplified after we remove all code to detect the default bridge network, right? If so, let's add a TODO comment for the   code to calculate the `sshdFirstNetwork` 🙏